.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "include/computecpp-sdk"]
+	path = include/computecpp-sdk
```

Comment:
Can I suggest putting this in modules/ or similar? It's not really just for includes, given that it has other code in it. I always envisioned users being able to set the include dir to `submodule-path/computecpp-sdk/include` so their code could `#include <vptr/virtual_pointer.hpp>` and that would be that.

Reply:
That's a really good point - especially given that we're now using the FindComputeCpp.cmake from the sdk.

Reply:
We typically use "external" for this

Reply:
ah, I pushed a commit moving this to 'modules' right as you posted this! I'll change it to external - it's not a big change.